scripts/test/run-tests.sh

```diff
@@ -17,7 +17,7 @@ WORKSPACE=./tests
 
 BBB_DEBIAN_SDCARD_IMAGE_URL="https://debian.beagleboard.org/images/bone-debian-10.3-iot-armhf-2020-04-06-4gb.img.xz"
 # Not on official home page, but found via https://elinux.org/Beagleboard:BeagleBoneBlack_Debian:
-BBB_DEBIAN_EMMC_IMAGE_URL="https://rcn-ee.com/rootfs/bb.org/testing/2020-09-21/buster-console/bone-debian-10.5-console-armhf-2020-09-21-1gb.img.xz"
+BBB_DEBIAN_EMMC_IMAGE_URL="https://rcn-ee.com/rootfs/bb.org/testing/2021-01-11/buster-console/bone-debian-10.7-console-armhf-2021-01-11-1gb.img.xz"
```

Comment:
This requires a change also in line `102`

Reply:
Right. This could be improved, but I won't spend time on that now, since we have https://tracker.mender.io/browse/QA-262.